dashboard/app/assets/javascripts/header.js

```diff
@@ -172,6 +172,45 @@ dashboard.buildHeader = function (stageData, progressData, currentLevelId, userI
   }
 };
 
+dashboard.initSendToPhone = function(selector) {
```

Comment:
I'd rather move this logic to the `dashboard.SendToPhone` component, but the jQuery mask plugin wants to manipulate the DOM.